src/index.js

```diff
@@ -1,10 +1,42 @@
 const express = require('express');
-const path = require('path');
 
-const app = express()
+const app = express();
+const cors = require('cors');
+const session = require('express-session');
 const port = process.env.SERVER_PORT || 5000;
+const { Pool } = require('pg');
+
+const pgSession = require('connect-pg-simple')(session);
+
+app.use(cors());
+
+app.use(session({
+    secret: 'mit tsl teacher moments',
```

Comment:
Are these secrets sensitive and should they be stored elsewhere (like environmental variables)? I know this is just a stub, but if it's sensitive, would be good to file a follow up issue for not saving this in the codebase.

Reply:
I kept it public because @gnarf 's implementation example showed that the secret was open. @gnarf can you comment on this?